pygmt/base_plotting.py

```diff
@@ -994,3 +994,353 @@ def text(
 
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("text", arg_str)
+
+    @fmt_docstring
+    @use_alias(R="region", J="projection", B="frame", C="offset")
+    @kwargs_to_strings(R="sequence",)
+    def meca(
+        self,
+        spec,
+        scale,
+        lon=None,
+        lat=None,
```

Comment:
Should we use `longitude` and `latitude` instead? My feeling is most Python package use long, descriptive argument names instead of short names.

Reply:
Thanks for pointing this out. Pushing this change now.